dev/integration_tests/display_cutout_rotation/lib/main.dart

```diff
@@ -0,0 +1,50 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
+final class MyApp extends StatefulWidget {
+  const MyApp({super.key});
+
+  @override
+  State<MyApp> createState() => _MyAppState();
+}
+
+class _MyAppState extends State<MyApp> {
+  @override
+  Widget build(BuildContext context) {
+    final List<DisplayFeature> displayFeatures =
```

Comment:
Let's move the `getCutouts` feature into this `lib/` and re-use it instead of having two copies.

Reply:
I didnt do this. It is 2 lines.